packages/api/cms-api/src/index.ts

```diff
@@ -131,6 +139,8 @@ export { OneToManyFilter } from "./common/filter/one-to-many.filter";
 export { StringFilter } from "./common/filter/string.filter";
 export { extractGraphqlFields } from "./common/graphql/extract-graphql-fields";
 export { CdnGuard } from "./common/guards/cdn.guard";
+export { getCrudSearchFieldsFromMetadata } from "./common/helper/curd-generator.helper";
+export { hasFieldFeature } from "./common/helper/curd-generator.helper";
```

Comment:
can't you move those internal helpers into the other package?

Reply:
Right now, I set up the packages that `@comet/cms-api` is the main dependency and should work standalone.

`@comet/api-generator` is thought to be an optionally useable cli tool and it is only responsible for generating propper source code.

The `@comet/cms-api` internally uses `getCrudSearchFieldsFromMetadata` for example in `redirects.service.ts`. 

I can not think of a propper way to bring this to `@comet/api-generator` package easily.

Reply:
It's already named `get`**Crud**`SearchFieldsFromMetadata`. I am not sure if it really needs more.

The only way that makes it (for me) more explicit is maybe moving the crud indication closer to Metadata - maybe something like: `getSearchFieldsFromCrudMetadata`

Reply:
I was talking about hasFieldFeature, sorry for the confusion

Reply:
ah yeah - makes sense ^^

Reply:
renamed `hasFieldFeature` to `hasCrudFieldFeature`